liboptv/tests/CMakeLists.txt

```diff
@@ -35,6 +35,11 @@ if (UNIX)
     set(LIBS ${LIBS} -lm -pthread)
 endif (UNIX)
 
+if (APPLE)
```

Comment:
I'm pretty sure this shouldn't be necessary. Perhaps you have two versions of liboptv installed, one in /usr/ and one in /usr/local/ ?

If so, I suggest to clean up the /usr/ version and then this hard-coded path will not be necessary.